tests/integration/test_oauth_state_init_endpoint.py

```
@@ -0,0 +1,945 @@
+                "redirect_uri": "https://myapp.example.com/oauth/callback"
+            }
+
+            # Act
+            response = test_client.post("/api/auth/gmail/init", json=request_body)
+
+            # Assert
+            assert response.status_code == 200
+            data = response.json()
+            assert data["success"] is True
+
+    def test_duplicate_state_token_overwrites_previous(self, test_client, mock_rate_limiter):
```

Ruff (0.14.10)

Unused method argument: `mock_rate_limiter`

(ARG002)